mod/consensus-types/pkg/types/fork.go

```diff
@@ -139,3 +139,15 @@ func (f *Fork) HashTreeRootWith(hh fastssz.HashWalker) error {
 func (f *Fork) GetTree() (*fastssz.Node, error) {
 	return fastssz.ProofTree(f)
 }
```

Comment:
Nit- add one of these separators before these read methods
`
/* -------------------------------------------------------------------------- */
/*                                   FastSSZ                                  */
/* -------------------------------------------------------------------------- */ 
`

Reply:
Sure, added.